CONTRIBUTING.md

```diff
@@ -50,11 +50,31 @@ include BREAKING CHANGE keyword for breaking changes.
 
 The message summary should be a one-sentence description of the change, and it must be 72 characters in length or shorter. For a list of tags, please [click here](https://github.com/conventional-changelog/commitlint/tree/master/%40commitlint/config-conventional#type-enum). We use the angular style for tags. See the [default release rules](https://github.com/semantic-release/commit-analyzer/blob/master/lib/default-release-rules.js) based on the commit tag. Shown below is an example of the release type that will be done based on a commit message.
 
-| Commit message                                                                                                                                          | Release type               |
-| ------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------- |
-| `fix(preview): fullscreen behavior in ie11`                                                                                                             | Patch Release              |
-| `feat(preview): add ability to fullscreen`                                                                                                              | ~~Minor~~ Feature Release  |
-| `feat(preview): remove fullscreen ability`<br><br>`BREAKING CHANGE: The fullscreen ability has been removed due to poor support of the fullscreen api.` | ~~Major~~ Breaking Release |
+#### Semantic Versioning
```

Comment:
This is documented elsewhere. What do you think about linking instead?
ref: https://docs.npmjs.com/about-semantic-versioning

Reply:
What do you mean by document elsewhere? By someone else or by us?

Reply:
I think we should be clear about our definitions even if we lean on an externally-maintained config at the moment (abstracted through our frontend config repo).

Reply:
adding a link to this page at the end